src/wp-admin/update-core.php

```diff
@@ -744,7 +744,7 @@ function list_theme_updates() {
 	<tr>
 		<td class="check-column">
 			<?php if ( $compatible_wp && $compatible_php ) : ?>
-				<label for="<?php echo $checkbox_id; ?>" class="label-covers-full-cell">
+				<label for="<?php echo $checkbox_id; ?>">
```

Comment:
The themes table still has the label first (it was for both plugins and themes).